lmnqlbridge/operator/toDate.go

```diff
@@ -0,0 +1,43 @@
+package operator
+
+import (
+	"cloud.google.com/go/civil"
+	"fmt"
+	"github.com/araddon/qlbridge/expr"
+	"github.com/araddon/qlbridge/value"
+)
+
+// todate:   convert to Date
+//
+//	// uses lytics/datemath
+//	todatetime("now-3m")
+//
+//	// uses araddon/dateparse util to recognize formats
+//	todatetime(field)
+//
+//	// first parameter is the layout/format
+//	todatetime("01/02/2006", field )
+type ToDate struct{}
+
+// Type time
```

Comment:
Don't think we need to add comments on this just to please the editor, also this should be "date" if you want to keep the comment, right? :)